template-widget-extension/package.json

```diff
@@ -1,91 +1,91 @@
-        "lumapps-sdk-js": "^1.0.1",
-        "moment": "^2.29.1",
-        "moment-range": "^4.0.2",
-        "react": "^16.13.1",
-        "react-dom": "^16.13.1",
-        "react-intl": "5.15.7",
-        "react-redux": "7.2.4",
-        "reselect": "4.0.0"
-    },
-    "devDependencies": {
-        "@lumapps-extensions-playground/devenv": "^1.3.0",
-        "@lumapps-extensions/shipping-server": "^1.1.3",
-        "@testing-library/jest-dom": "^4.2.4",
-        "@testing-library/react": "^9.3.2",
-        "@testing-library/user-event": "^7.1.2",
-        "@types/enzyme": "^3.10.8",
-        "@types/enzyme-adapter-react-16": "^1.0.6",
-        "@types/jest": "^24.0.0",
-        "@types/lodash": "^4.14.168",
-        "@types/node": "^12.0.0",
-        "@types/react": "^16.9.0",
-        "@types/react-dom": "^16.9.0",
-        "@typescript-eslint/eslint-plugin": "^2.18.0",
-        "@typescript-eslint/parser": "^2.18.0",
-        "enzyme": "^3.11.0",
-        "enzyme-adapter-react-16": "^1.15.6",
-        "enzyme-to-json": "^3.6.1",
-        "eslint": "^6.8.0",
-        "eslint-config-airbnb": "^18.0.1",
-        "eslint-config-prettier": "^6.10.0",
-        "eslint-plugin-import": "^2.20.0",
-        "eslint-plugin-jsx-a11y": "^6.2.3",
-        "eslint-plugin-prettier": "^3.1.2",
-        "eslint-plugin-react": "^7.18.0",
-        "eslint-plugin-react-hooks": "^2.3.0",
-        "jest": "^24.9.0",
-        "npm-run-all": "^4.1.5",
-        "prettier": "2.2.0",
-        "prettier-stylelint": "^0.4.2",
-        "react-scripts": "3.4.3",
-        "stylelint": "^9.10.1",
-        "stylelint-config-idiomatic-order": "^6.2.0",
-        "stylelint-config-recommended": "^2.1.0",
-        "stylelint-config-recommended-scss": "^3.2.0",
-        "stylelint-scss": "^3.5.4",
-        "ts-loader": "8.1.0",
-        "typescript": "^4.0.3"
-    }
-}
+  "name": "template-react-ts",
+  "version": "0.4.4",
+  "private": true,
+  "scripts": {
+    "check": "tsc --noEmit",
+    "check:typing": "yarn run check",
+    "start": "run-p start:playground start:shipping-server",
+    "start:playground": "react-scripts start",
+    "start:shipping-server": "lumapps-shipping-server",
+    "test": "react-scripts test --coverage"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "jest": {
+    "snapshotSerializers": [
+      "enzyme-to-json/serializer"
+    ],
+    "transformIgnorePatterns": [
+      "node_modules/(?!@lumx/icons|@lumx/react)"
+    ]
+  },
+  "dependencies": {
+    "@lumapps-extensions-playground/common": "^1.3.0",
+    "@lumx/core": "^2.0.1",
+    "@lumx/icons": "^2.0.1",
+    "@lumx/react": "^2.0.1",
+    "axios": "0.21.1",
+    "lodash": "4.17.21",
+    "lumapps-sdk-js": "^1.0.0",
```

Comment:
This version is bugged, please use the previous one (0.0.44 as far as i remember)